src/ui/db/zcl_abapgit_gui_page_db.clas.abap

```diff
@@ -22,29 +22,55 @@ CLASS zcl_abapgit_gui_page_db DEFINITION
         delete  TYPE string VALUE 'delete',
         backup  TYPE string VALUE 'backup',
         restore TYPE string VALUE 'restore',
-      END OF c_action .
+      END OF c_action.
+
+    DATA mt_methods TYPE zcl_abapgit_background=>ty_methods.
 
     CLASS-METHODS backup
       RAISING
-        zcx_abapgit_exception .
+        zcx_abapgit_exception.
```

Comment:
no need to change this whitespace, next person changing the class in SE24 will roll back this exact change